vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "System.Management.Automation": "6.1.7601.17515",
```

Comment:
Ahh, it is needed for PowerShell in the tests. https://msdn.microsoft.com/en-us/library/system.management.automation(v=vs.85).aspx

PowerShell is open source and can run in multi-platform now: https://azure.microsoft.com/en-us/blog/powershell-is-open-sourced-and-is-available-on-linux/

So maybe we got a shot there ?